VEHICLE/SetTransformRateForAnimation.md

```diff
@@ -0,0 +1,41 @@
+---
+ns: VEHICLE
+aliases: ["0x498218259FB7C72D", "_SET_UNK_FLOAT_0x104_FOR_SUBMARINE_VEHICLE_TASK"]
+---
+## SET_TRANSFORM_RATE_FOR_ANIMATION
+
+```c
+// 0x498218259FB7C72D
+void SET_TRANSFORM_RATE_FOR_ANIMATION(Vehicle vehicle, float transformRate);
+```
+
+### Affects the playback speed of the submarine car conversion animations. Does not affect hardcoded animations such as the wheels being retracted. In decompiled scripts the only value used for transformRate is 2.5.
```

Comment:
Using triple hashes `#`, makes the text look huge, maybe get rid of them, not needed for descriptions. Thanks!